### DISCUSSION TITLE
Appropriate cases to use eager loading

### DISCUSSION THREAD

**Discussion by LauraBeatris** [vercel/next.js#18792] [Help]
What are the appropriate cases to use eager loading? It's not something specific to Next but I would like to understand more about the causes of it. 

As it would make the image be loaded instantly - I'm using it on the first elements of a page, such as Headers. 

Is it correct? What are the other cases?

**Comment by LauraBeatris** [comment]
And also, I don't know what are the cases to preload images using the ``priority`` property

**Reply by DoctorDerek** [reply]
@LauraBeatris According to this https://nextjs.org/docs/api-reference/next/image#layout then eager shouldn't be used in favor of priority. 

You should preload anything that is above the fold (starts in the viewport) according to https://web.dev/preload-responsive-images/ -- like the first slide on an image slideshow.

**Reply by LauraBeatris** [reply]
Got it! Thanks
